File: Easy/JAVA/CaesarCipherEncryptor.java
public class CaesarCipherEncryptor {

    final static int ENGLISH_ALPHABET = 26;

    public static String Encrypt(String str, int key) {
        StringBuilder output = new StringBuilder();

        for (int i = 0; i < str.length(); i++) {
            if (Character.isLowerCase(str.charAt(i))) {
                int character = (str.charAt(i) - 'a' + key) % ENGLISH_ALPHABET + 'a';
                char ch = (char) (character);
                output.append(ch);
            } else {
                int character = (str.charAt(i) - 'A' + key) % ENGLISH_ALPHABET + 'A';
                char ch = (char) (character);
                output.append(ch);
            }
        }
        
        return output.toString();
    }

    public static void main(String[] args) {
        String str = "shajklxyz";
        String result = Encrypt(str, 10);
        System.out.println(result);
    }

}

File: Easy/PHP/CaesarCipherEncryptor.php
<?php

namespace Easy\PHP\CaesarCipherEncryptor;

class Encrypt
{
    public $ENGLISH_ALPHABET = 26;
    public $output;

    public function encrypt($str, $key)
    {

        for ($i = 0; $i < strlen($str); $i++) {
            if (ctype_lower($str[$i])) {
                $character = (ord($str[$i]) - ord('a') + $key) % $this->ENGLISH_ALPHABET + ord('a');
                $this->output .= chr($character);
            } else {
                $character = (ord($str[$i]) - ord('A') + $key) % $this->ENGLISH_ALPHABET + ord('A');
                $this->output .= chr($character);
            }
        }

        return $this->output;
    }
}


$str = "shajklxyz";
$key = 2;
$encrypt  = new Encrypt();
$result = $encrypt->encrypt($str, $key);
print_r($result);
